writing/kill-the-dog.md

```diff
@@ -0,0 +1,25 @@
+# Killing the Dog - My new reason for practicing mindfulness
+
+Anyone else seriously wanted to kill their dog ☠️🐶? And gone through a bunch of plans to make it seem like an accident?
+
+- "Accidentally" leaving the front door open so she runs into the road
+- Host a house party 🥳, put on "Who let the dogs out", turn it up and keep putting the puppy in the middle of the dance floor until someone crushes her to death with their funky moves
+- Travel back to the Roman times, tell everyone the dog is the Messiah and wait until the inevitable crucifixion ✟
+
+Yeah, I thought about this. In the first few weeks of getting the most adorable little puppy I found myself in a permanent state where I couldn't think clearly for the first time in my life. Possibly due to a mixture of sleep deprivation, constant distraction, worrying and having massively underestimated the commitment.
+
```

Comment:
Delete "having massively underestimated the commitment" and replace with something lighter per our discussion.  The way you have it starts to stray from the theme into another one...